examples/ts/sui/get-transaction-hex.ts

```diff
@@ -21,29 +21,33 @@ const accessToken = '';
 // TODO: set a walletId
 const walletId = '';
 
-// TODO: set your passphrase for your wallet here
-const passphrase = '';
-
 async function buildPayTx() {
-  bitgo.authenticateWithAccessToken({ accessToken });
-  const newWallet = await bitgo.coin(coin).wallets().getWallet({ id: walletId });
-  const recipients = [
-    {
-      address: 'tsui-address', // TODO: set a tsui receive address
-      amount: '10000', // TODO: set an amount for the transaction
-    },
-  ];
-
-  const response = await newWallet.sendMany({
-    recipients,
-    type: 'transfer',
-    walletPassphrase: passphrase,
-  });
-  const tx = response.tx;
-  const requiredTx = tx.replace(/\+/g, '%2B').replace(/\//g, '%2F').replace(/=/g, '%3D');
-
-  console.log('Required tx param :');
-  console.log(requiredTx);
+  try {
+    bitgo.authenticateWithAccessToken({ accessToken });
+    const wallet = await bitgo.coin(coin).wallets().getWallet({ id: walletId });
+    const recipients = [
+      {
+        address: 'tsui-address', // TODO: set a tsui receive address
+        amount: '10000', // TODO: set an amount for the transaction
+      },
+    ];
+
+    const transaction = await wallet.prebuildTransaction({
+      recipients,
+      type: 'transfer',
+    });
```

Comment:
without the preview flag, its a txRequest creation
```suggestion
    const transaction = await wallet.prebuildTransaction({
      recipients,
      type: 'transfer',
      preview: true,
    });
```

Reply:
maybe it needs to be a different example file, this one seems to be used to convert from base64 to hex

Reply:
Hi @alebusse, this example file was created by me, it is used to generate a hex which is required to be passed in the api for fee estimate for sui. The conversion is done as prebuildTransaction returns tx in a format which sui doesn't expect in its sui_dryRunTransactionBlock endpoint